warehouse/models/mart/gtfs/_mart_gtfs.yml

```diff
@@ -1453,3 +1496,130 @@ models:
       - name: gtfs_dataset_key
         description: |
           The primary key for the record in `dim_gtfs_datasets` associated with this feed.
+  - name: fct_daily_scheduled_shapes
+    description: |
+      A daily table showing shapes that had trips scheduled on a given date.
+    columns:
+      - name: key
+        description: |
+          Synthetic primary key constructed from `service_date`, `shape_id`, and `shape_array_key`.
+        tests:
+          - unique:
+              where: "not contains_warning_duplicate_trip_primary_key"
+          - not_null
+      - name: n_trips
+        description: The number of trips associated with this shape on the given date.
+      - name: feed_key
+        description: |
+          Foreign key to the `dim_schedule_feeds` table.
+      - name: service_date
+        description: Date on which this shape was active.
+      - name: shape_id
+        description: '{{ doc("gtfs_trips__shape_id") }}'
+      - name: shape_array_key
+        description: Foreign key to dim_shapes_arrays.
+      - name: pt_array
+        description: Ordered array of WKT points that describe this shape.
+      - name: contains_warning_duplicate_trip_primary_key
+        description: |
```

Comment:
See comment on `stops` -- because we are *aggregating* trips here, this should be refactored to explain the impact that this warning has on the counts. Rows with `true` here may have inflated `n_trips` because of duplicated primary keys.